etsin_finder/frontend/js/components/qvain/files/dataCatalog.jsx

```diff
@@ -66,16 +81,41 @@ class DataCatalog extends Component {
           options={availableOptions}
           onChange={(selection) => {
             setDataCatalog(selection.value)
-            this.setState({ errorMessage: undefined })
+            this.setState({
+              errorMessage: undefined,
+              fileOrigin: selection.label,
+            })
           }}
           onBlur={this.handleOnBlur}
           attributes={{ placeholder: 'qvain.files.dataCatalog.placeholder' }}
           isDisabled={(selected.length > 0) || (original !== undefined) || isPas}
         />
+        <DoiSelectionContainer>
+          <Checkbox
+            id="doiSelector"
+            style={{ display: (this.state.fileOrigin === 'IDA' && original === undefined) ? 'inline-block' : 'none' }}
```

Comment:
I would advise to not use inline styles if possible: https://reactjs.org/docs/faq-styling.html#are-inline-styles-bad